### DISCUSSION TITLE
Prima versione del nuovo frontend

### DISCUSSION THREAD

**Discussion by pdonadeo** [pdonadeo/e-uscito-joypad#40] [Announcements]
Frontend completamente rinnovato, con la possibilità di cercare un gioco all'interno degli episodi.

<hr /><em>This discussion was created from the release <a href='https://github.com/pdonadeo/e-uscito-joypad/releases/tag/v9'>Prima versione del nuovo frontend</a>.</em>

**Comment by pdonadeo** [comment]
In modo del tutto **non** democratico ed autoritario — _ma lo faccio per voi, per abituarvi a quello che vi aspetta ad Ottobre_ — ho deciso che il frontend fatto da @MarcoJul era quasi perfetto e ho deciso di andare on line, con qualche piccola modifica.

Quindi in questo momento la versione [pubblica del sito](https://www.euscitojoypad.it/) e quella di [staging](https://staging.euscitojoypad.it/) sono identiche.

Ci sono certamente aspetti da migliorare ma lo faremo al ritorno dalle vacanze. In questo mese di noia videoludica mi sembrava carino mettere a disposizione degli ascoltatori di Joypad uno strumento per cercare tra le centinaia di titoli di cui si è parlato nel programma.

È imperfetto? Sì. È peggio di Cyberpunk 2077 all'uscita? No!

@zampale facci sapere se ti piace 😄

**Reply by agofa** [reply]
E non assomiglia neanche un po' a star Citizen. Soprattutto nel budget

**Reply by MarcoJul** [reply]
Eccomi, scusate ma ieri ero lontano dal computer e ho visto la rivoluzione solo stamattina😅

Tutto bene! Purtroppo avevo lasciato il codice nel fango fra commenti/codice non utilizzato e affini che però ho visto che per fortuna @pdonadeo si è messo a fare le pulizie✌️

A parte le issue, a me mancava da inserire il funzionamento giusto dell'ordinamento su desktop e la pagina di info del gioco, sempre su desktop, in questi giorni la sistemo così nel caso, appena possibile, può venire aggiornata direttamente!

**Reply by bionicco** [reply]
Grandi! Durante l'estate mi sono un po' disconnesso e mi ero perso l'aggiornamento, scoprendolo solo aprendo per caso il sito!
Invece è stato fatto il grande passo e finalmente tutti potranno andare a ritrovare in quale momento di joypad si è parlato di un determinato titolo (e chissà se ero l'unico che sentiva questa esigenza :D) 
Come ogni progetto è ancora perfezionabile, ma mi sembra assolutamente usabile e completo, quindi grazie a tutti

**Comment by zampale** [comment]
È splendido, posso dire solo questo.
CD Projekt se lo sogna un lancio così.

Il 31 lug 2022, 12:00 +0200, Agostino Fanti ***@***.***>, ha scritto:

**Comment by zampale** [comment]
Tra l’altro, volete farmi una lista di credits così vi cito tutti?
Il 31 lug 2022, 12:00 +0200, Agostino Fanti ***@***.***>, ha scritto:

**Reply by pdonadeo** [reply]
Te la faccio questa sera, sto uscendo ora.

**Reply by MarcoJul** [reply]
Mi inserisco giusto per dare il giusto credito a [Chiara](https://www.instagram.com/chiararicolfi/) e alla sua UI, senza di lei avrei messo a frontend roba tipo pulsantoni viola su fondo nero stile anni 90😅

**Comment by pdonadeo** [comment]
Devo ammettere una cosa: non ho voglia di spulciare centinaia di messaggi per recuperare nomi, cognomi  e non che altre informazioni di tutti per i credits.

Quindi lancio un appello: scrivete in questo thread nome, cognome e una URL (ad un social, un blog, un sito, quello che vi pare). Poi li raccolgo tutti in una lista comoda per @zampale.

Inizio io: Paolo Donadeo https://github.com/pdonadeo (non ho nessun altro profilo social)

**Reply by agofa** [reply]
Agostino Fanti https://github.com/agofa (non so cosa altro mettere, frequento poco o nulla i social)

**Reply by MarcoJul** [reply]
Marco Giuliotti [Instagram](https://www.instagram.com/marcojull/) o [Github](https://github.com/MarcoJul)

Chiara Ricolfi [Instagram](https://www.instagram.com/chiararicolfi/)

**Reply by bionicco** [reply]
Niccolò Biondi [Instagram](https://www.instagram.com/bionicco/)
